nilearn/glm/tests/test_utils.py

```diff
@@ -31,12 +37,108 @@ def test_full_rank():
 
 
 def test_z_score():
-    p = np.random.RandomState(42).uniform(size=10)
-    assert_array_almost_equal(norm.sf(z_score(p)), p)
-    # check the numerical precision
-    for p in [1.e-250, 1 - 1.e-16]:
-        assert_array_almost_equal(z_score(p), norm.isf(p))
-    assert_array_almost_equal(z_score(np.float32(1.e-100)), norm.isf(1.e-300))
+    # ################# Check z-scores computed from t-values #################
+    # Randomly draw samples from the standard Student’s t distribution
+    tval = np.random.standard_t(1e10, size=10)
```

Comment:
Why do you use 1e10 dofs ? this amounts to rendering the t distribution identical to a normal distrbution...

Reply:
Ups, yes. I changed that to 10. I've also changed the between- and within- group dof's of the F-distribution in line 68.